lib/src/widgets/confirmation_dialog.dart

```diff
@@ -28,23 +30,20 @@ Future<void> showConfirmationDialog(
         constraints: const BoxConstraints(maxWidth: 500),
         child: Row(
           children: [
-            const Icon(
-              YaruIcons.question,
-              size: 64.0,
-            ),
+            if (icon != null) icon,
             const SizedBox(width: 16),
```

Comment:
do we want the spacing if there's no icon or should it be inside the icon != null check?

Reply:
Good catch, didn't see that - the space should only be there if there's also an icon.